omniflixhub/assetlist.json

```diff
@@ -39,6 +39,37 @@
       },
       "type_asset": "sdk.coin"
     },
+    {
+      "description": "GATA Yield DAO Governance Token powered by OmniFlixHub",
+      "denom_units": [
+        {
+          "denom": "factory/omniflix1fwphj5p6qd8gtkehkzfgac38eur4uqzgz97uwvf6hsc0vjl004gqfj0xnv/ygata",
+          "exponent": 0
+        },
+        {
+          "denom": "ygata",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/omniflix1fwphj5p6qd8gtkehkzfgac38eur4uqzgz97uwvf6hsc0vjl004gqfj0xnv/ygata",
+      "name": "Yield GATA",
+      "display": "yGATA",
+      "symbol": "yGATA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/omniflixhub/images/ygata.png",
```

Comment:
these images are not identical because one has a white background while the other has a transparent background. Please ensure both svg and png of the logo_URIs and an image object have the same appearance.

Reply:
It looks like you've made changes, but the images are still different (both in padding and background)